live-build/misc/ansible-roles/appliance-build.minimal-common/tasks/main.yml

```diff
@@ -544,3 +544,13 @@
     state: directory
     mode: 0755
     recurse: yes
+
+#
+# Setup tmpfs for /tmp
```

Comment:
this comment isn't helpful; we should either remove it, or have it complement the code by explaining _why_ we want `/tmp` to be `tmpfs`.